project/frontend/ssl_setup.bat

```diff
@@ -0,0 +1,4 @@
+choco install mkcert
+mkcert -install
+mkcert localhost
+set HTTPS=true&&set SSL_CRT_FILE=localhost.pem&&set SSL_KEY_FILE=localhost-key.pem&&npm start
```

Comment:
What exactly does this line set in my env variables? @ChandlerCabrera I'm getting an error after running this script saying 
> "You specified SSL_CRT_FILE in your env, but the file "C:\Users\Ericn\Documents\GitHub\capstone-project\project\frontend\localhost.pem" can't be found."

Reply:
The *mkcert localhost* command should be creating a localhost.pem file in the frontend directory. Is that file missing for you?

Reply:
That line enables HTTPs, specifies the certification file as localhost.pem (should be located in frontend folder), and specifies the key file as localhost-key.pem

Reply:
Yes, Im not having any created when it runs that script. (I'm testing it in windows) @digengill did you run into any similar issues when you tried the SSL script?

Reply:
I will try this setup again from scratch and see if I run into any issues